djhtml/modes.py

```diff
@@ -287,9 +287,19 @@ def create_token(self, raw_token, src, line):
 
         if raw_token == "<":
             if tag := re.match(r"([\w\-\.:]+)", src):
-                token, mode = Token.Text(raw_token, mode=DjHTML), InsideHTMLTag(
-                    tag[1], line, self
+                tagname = tag[1]
+                absolute = True
+                token = Token.Text(raw_token, mode=DjHTML)
+                offsets = dict(
+                    relative=-1 if line.indents else 0,
+                    absolute=len(line) + len(tagname) + 2,
                 )
+                if tag := re.match(tagname + r"[ \t]*\n", src):
+                    # Use "relative" multi-line indendation instead
+                    absolute = False
+                    token.indents = True
```

Comment:
Admittedly, this is "cheating" a bit because it changes an "internal" attribute of a token has already been created. It saves a whole bunch of if/else statements though, especially further down where the `>` token is created.